scripts/scalafmt

```diff
@@ -3,7 +3,7 @@
 set -e
 
 HERE="`dirname $0`"
-VERSION=$(sed -nre "s#\s*version[^0-9]+([0-9.]+)#\1#p" $HERE/../.scalafmt.conf)
+VERSION=$(sed -nre "s#version[^0-9]+([0-9.]+)#\1#p" $HERE/../.scalafmt.conf)
```

Comment:
I'm curious & for posterity trying to figure this change out: Why does deleting matching optional 
whitespace make things work on FreeBSD (and is
not required on Linux, etc.).

There is a group of people who like optional whitespace and another which vehemently oppose it.   In the days of punched cards, whitespace was expensive.

Reply:
Using `\s` to skip whitespaces doesn't seem portable. To keep the behavior I can replace it with `[[:space:]]`, i.e. to `"s#[[:space:]]*version[^0-9]+([0-9.]+)#\1#p"`

Reply:
```
% sed -nre "s#\s*version[^0-9]+([0-9.]+)#\1#p" .scalafmt.conf
sed: 1: "s#\s*version[^0-9]+([0- ...": RE error: trailing backslash (\)

% sed -nre "s#[[:space:]]*version[^0-9]+([0-9.]+)#\1#p" .scalafmt.conf
3.7.14
```

Reply:
I like the "[[:space:]]" approach.   

Do you have a Windows system?  If yes, does it work there?  If no, I can dust of my Windows system, find 
its mouse & keyboard, fire it up, and try it there.

Reply:
It works on gnu sed, and so it seems to work on windows, too.